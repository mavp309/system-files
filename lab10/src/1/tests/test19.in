i 100
i 50
i 75
p -1
d 50
s 75
r -1
p -1
q 0
