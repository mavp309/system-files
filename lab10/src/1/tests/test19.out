75 50 100 
Deleted
True
100 75 
